src/sqlfluff/core/config.py

```diff
@@ -542,7 +542,7 @@ def __getstate__(self):
         del state["_plugin_manager"]
         return state
 
-    def __setstate__(self, state):
+    def __setstate__(self, state):  # pragma: no cover
```

Comment:
AFAIK, this is unrelated to the rest of the PR, but it was coming up in the PR build as "not covered". 🤷‍♀️

Reply:
FYI I got this, then it fixed itself while I was digging in codecov. So I suspect it might not need it again. But could be wrong!

```suggestion
    def __setstate__(self, state):
```